src/EventStore.Client.Streams/EventStoreClient.Delete.cs

```diff
@@ -12,77 +12,65 @@ public partial class EventStoreClient {
 		/// </summary>
 		/// <param name="streamName">The name of the stream to delete.</param>
 		/// <param name="expectedRevision">The expected <see cref="StreamRevision"/> of the stream being deleted.</param>
+		/// <param name="deadline">The maximum time this operation is allowed to take.</param>
 		/// <param name="configureOperationOptions">An <see cref="Action{EventStoreClientOperationOptions}"/> to configure the operation's options.</param>
 		/// <param name="userCredentials">The optional <see cref="UserCredentials"/> to perform operation with.</param>
 		/// <param name="cancellationToken">The optional <see cref="System.Threading.CancellationToken"/>.</param>
 		/// <returns></returns>
 		public Task<DeleteResult> SoftDeleteAsync(
 			string streamName,
 			StreamRevision expectedRevision,
+			TimeSpan? deadline = null,
 			Action<EventStoreClientOperationOptions>? configureOperationOptions = null,
```

Comment:
probably remove this argument now

Reply:
how sorry?

Reply:
Hmm you are right, `DeleteResult` is not an `IWriteResult` (although maybe it should be)